services/distribution/src/test/resources/application.yaml

```diff
@@ -195,6 +195,8 @@ services:
       allow-list: ${DCC_VALIDATION_SERVICE_ALLOWLIST:{"certificates":[{"serviceProvider":"ACME Inc.","hostname":"validation-service.acme.org","fingerprint256":"E87E26EB9D22280125E113F755BE2F9F90DB076DD011D5940E0A78DD2F20EDFA"}],"serviceProviders":[{"serviceProviderAllowlistEndpoint":"https://validation-service.acme.org/serviceproviders","fingerprint256":"E87E26EB9D22280125E113F755BE2F9F90DB076DD011D5940E0A78DD2F20EDFA"}]}}
       allow-list-signature: ${DCC_VALIDATION_SERVICE_ALLOWLIST_SIGNATURE:3046022100945f80f1fa7b32cc9f4f951ec826fab13274472299089cc980448d66ff4e3055022100917dda6bedb1ac209715352b6355142fc23a0e5d5e5743559e3813be54a1b0b8}
       allow-list-certificate: ${DCC_VALIDATION_SERVICE_ALLOWLIST_CERTIFICATE:MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEMMvW0zun8fNCELK1tsqXsGJPu4p7850ZPCBCoxQ5gs2z5G0in3izL7eTFa5lI7Gkhnz0tN5whVQJObCaqbP55A==}
+      ccl-directory: ${COMMON_COVID_LOGIC:ccl}
+      ccl-allow-list: CCL-DE-0001
```

Comment:
list? can we please test with two or more items?

Reply:
yes, sure :)

Reply:
yes